composer/utils/iter_helpers.py

```diff
@@ -88,3 +89,37 @@ def iterate_with_pbar(iterator, progress_bar=None):
             yield x
             if pb is not None:
                 pb.update(len(x))
+
+
+class IteratorFileStream(io.RawIOBase):
```

Comment:
I'm a bit confused; I thought this was already in the other PR?

Reply:
Yeah - not sure why it's here, I merged from the other branch before it got merged into dev.